Libraries/Components/TextInput/TextInput.js

```diff
@@ -492,6 +492,12 @@ const TextInput = React.createClass({
     inlineImagePadding: PropTypes.number,
 
     /**
+     * Allow TextInput to pass touch event to parent.
```

Comment:
I don't understand this comment. Could you extend it please explaining how and why someone would want to use this prop?